### ISSUE TITLE
Error printed to CSV with flag experimentalSessionAndOrigin on 10.10.0

### ISSUE THREAD

**Issue by MateuszKobiera** [cypress-io/cypress#24302]
### Current behavior

I am testing downloading and editing csv in one test and in a new test I wanted to do some POST request on different domain, so I turn on flag experimentalSessionAndOrigin and then in test with csv I have printed error inside csv. 
Without this flag everything works fine. 
In this csv there are mentioned flags modifyObstructiveThirdPartyCode and modifyObstructiveCode.
I turn back to version 10.7.0 as there was some fix connected to that on 10.8.0 https://github.com/cypress-io/cypress/issues/22146
and again it works fine. 
I attached this csv
[CSV_with_error.csv](https://github.com/cypress-io/cypress/files/9818283/CSV_with_error.csv)

### Desired behavior

I would like to not print any error in my csv file.

### Test code to reproduce

I tried to reproduced it with some sample csv downloads, but there error is not print out. 
Basically it is sth like this:
`cy.visit("/download-sample-csv.php")
 cy.get("[download=\"SampleCSVFile_2kb.csv\"]").click()`
 and in downloads folder there is file with this error printed out 

### Cypress Version

10.10.0

### Node version

v16.17.1

### Operating System

Windows 11 22H2 22621.674

**Comment by AtofStryker** [comment]
Hi @MateuszKobiera . Thank you for opening an issue. This has to do with Cypress injecting multi domain into the CSV when it shouldn't. This is a duplicate of #24297 and going to close in favor of that issue.

**Comment by MateuszKobiera** [comment]
Hi @AtofStryker Ok, thank you!
